app/pages/home-for-user/news-pullout.jsx

```diff
@@ -101,6 +101,10 @@ const NewsSection = React.createClass({
   render() {
     const projLink = `${window.location.origin}/projects/${this.state.newestProject.slug}`;
     const avatarSrc = !!this.state.newestAvatar ? this.state.newestAvatar.src : null;
+    const recentProjects = 
+      this.props.updatedProjects
+      .sort((a, b) => { return new Date(b.updated_at) - new Date(a.updated_at); })
```

Comment:
You want the projects that were recently updated by their owners here, not the projects that you have most recently classified on, so the list has to be sorted by `project.updated_at` rather than `preference.updated_at`.